src/domains/transaction/validations/DelegateRegistration.ts

```diff
@@ -2,38 +2,30 @@ import { Networks } from "@ardenthq/sdk";
 import { selectDelegateValidatorTranslation } from "@/domains/wallet/utils/selectDelegateValidatorTranslation";
 import { validatePattern } from "@/utils/validations";
 
-export const delegateRegistration = (t: any, currentNetwork?: Networks.Network) => ({
-	username: (usernames: string[]) => {
-		if (currentNetwork === undefined) {
-			throw new Error(
-				"You need to provide a network to the `useValidation` hook to use the delegateRegistration.username validation.",
-			);
-		}
-
-		return {
-			maxLength: {
-				message: t("COMMON.VALIDATION.MAX_LENGTH", {
-					field: selectDelegateValidatorTranslation({
-						delegateStr: t("COMMON.DELEGATE_NAME"),
-						network: currentNetwork,
-						validatorStr: t("COMMON.VALIDATOR_NAME"),
-					}),
-					maxLength: 20,
-				}),
-				value: 20,
-			},
-			required: t("COMMON.VALIDATION.FIELD_REQUIRED", {
+export const delegateRegistration = (t: any) => ({
+	username: (usernames: string[], network: Networks.Network) => ({
+		maxLength: {
+			message: t("COMMON.VALIDATION.MAX_LENGTH", {
 				field: selectDelegateValidatorTranslation({
 					delegateStr: t("COMMON.DELEGATE_NAME"),
-					network: currentNetwork,
+					network,
 					validatorStr: t("COMMON.VALIDATOR_NAME"),
```

Comment:
since this validation rule is limited to ARK (we have a validator-specific one for mainsail), we can actually remove the validator translations completely